pkg/proxy/integrations/genericParser/util.go

```diff
@@ -4,24 +4,23 @@ import (
 	"encoding/base64"
 	"fmt"
 
-	// "fmt"
 	"unicode"
 
 	"go.keploy.io/server/pkg/hooks"
 	"go.keploy.io/server/pkg/models"
 	"go.keploy.io/server/pkg/proxy/util"
+	"go.uber.org/zap"
 )
 
-func PostgresDecoder(encoded string) ([]byte, error) {
+func PostgresDecoder(logger *zap.Logger, encoded string) []byte {
```

Comment:
@shivamsouravjha Why is this not returning the error?

Reply:
the error exported isn't used anywhere, I asked @Sarthak160 as well as how to proceed, as to use the error and take necessary steps or just simply log it.

Reply:
the error exported from this decoder isn't used in generic parser, had a discussion with @Sarthak160 which concluded in removing the unused return error

Reply:
if the consumer is not using the error, we should make the consumer handle the error - not require the entire entirely.